menus/context.cson

```diff
@@ -10,7 +10,7 @@
     ]
   ]
 
-  '.tree-view.full-menu .header.list-item':[
+  '.tree-view.full-menu .header.list-item':[ to '.tool-panel.tree-view .header.list-item':[
```

Comment:
Remove 'to' from the line ;) Should be `'.tool-panel.tree-view .header.list-item':[`